components/infobox/wikis/formula1/infobox_person_player_custom.lua

```diff
@@ -100,6 +102,7 @@ end
 ---@param personType string
 ---@return table
 function CustomPlayer:adjustLPDB(lpdbData, args, personType)
+	lpdbData.extradata.abbreviations = args.abbreviations
```

Comment:
Is this used somewhere?
If Yes, can we use `alternateid` instead of extradata?

Reply:
Where will it be used?

Reply:
i mentioned that `abbreviations` is not a valid field in `lpdb_player`
did not suggest to store it at all